core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2178,23 +2180,36 @@ class WorkspaceService(protected val userInfo: UserInfo,
     )
   }
 
-  private def syncPolicies(workspaceId: String, policyEmailsByName: Map[SamResourcePolicyName, WorkbenchEmail], workspaceRequest: WorkspaceRequest, parentSpan: Span) = {
-    traceWithParent("traversePolicies", parentSpan) (s1 =>
-      Future.traverse(policyEmailsByName.keys) { policyName =>
-        if (policyName == SamWorkspacePolicyNames.projectOwner && workspaceRequest.authorizationDomain.getOrElse(Set.empty).isEmpty) {
-          // when there isn't an auth domain, we will use the billing project admin policy email directly on workspace
-          // resources instead of synching an extra group. This helps to keep the number of google groups a user is in below
-          // the limit of 2000
-          Future.successful(())
-        } else if (WorkspaceAccessLevels.withPolicyName(policyName.value).isDefined || policyName == SamWorkspacePolicyNames.canCompute) {
-          // only sync policies that have corresponding WorkspaceAccessLevels to google because only those are
-          // granted bucket access (and thus need a google group)
-          traceWithParent(s"syncPolicy-${policyName}", s1)(_ => samDAO.syncPolicyToGoogle(SamResourceTypeNames.workspace, workspaceId, policyName))
-        } else {
-          Future.successful(())
-        }
+  private def syncPolicies(workspaceId: String, policyEmailsByName: Map[SamResourcePolicyName, WorkbenchEmail], workspaceRequest: WorkspaceRequest, parentSpan: Span): Future[List[Any]] = {
```

Comment:
If you want to, you can simplify the return type that you. Just `Future[Unit]` using `traverse_` instead of `traverse`